packages/solutions/app-tools/src/analyze/generateCode.ts

```diff
@@ -204,6 +208,21 @@ export const generateCode = async (
           await fs.writeFile(routesServerFile, code);
         }
 
+        const serverLoaderCombined = templates.ssrLoaderCombinedModule(
+          entrypoints,
+          entrypoint,
+          config,
+          appContext,
+        );
+        if (serverLoaderCombined) {
+          const serverLoaderFile = getServerCombinedModueFile(
+            internalDirectory,
+            entryName,
+          );
+          await fs.ensureFile(serverLoaderFile);
+          await fs.writeFile(serverLoaderFile, serverLoaderCombined);
```

Comment:
We can use `fs.outputFile` instead of `fs.ensureFile` + `fs.writeFile`

Reply:
Got it

Reply:
Done~